pontoon/contributors/templates/contributors/includes/timeline.html

```diff
@@ -1,24 +1,51 @@
 <h3>{{ contribution_timeline.title }}</h3>
 
-{% for title, values in contribution_timeline.contributions.items() %}
-<div class="contribution-group">
-    <h4 class="heading">
-        <span class="icon far {{ values.type }}"></span>
-        {{ title }}
-    </h4>
-    <ul class="localizations">
-        {% for _, localization in values.data.items() %}
-        <li>
-            <a href="{{ localization.url }}">
-                <span class="stress">{{ localization.project.name }}</span> &middot; {{
-                localization.locale.name }} <span class="stress">{{ localization.locale.code }}</span>
-                <span class="contribution-count">{{ ", ".join(localization.actions) }}</span>
-            </a>
-        </li>
+{% if contribution_timeline.year_shown %}
+    {% for month, month_data in contribution_timeline.contributions.items() %}
+    <div class="contribution-year">
+        <span class="month-label">{{ month }}</span>
+        {% for title, project_data in month_data.items() %}
+        <div class="contribution-group">
```

Comment:
It looks like this element is repeated exactly below? Could it be defined once and used twice? DRY isn't an absolute rule, but for something this complex it should be followed unless there's a good reason not to.